hack/run-locally.sh

```diff
@@ -94,24 +121,184 @@ function wait_for_cluster() {
     fi
 }
 
+function print_usage() {
+    >&2 echo "Usage: $0 [mode] -c <cluster-dir> [options]
+
+$0 can be run in different modes:
+
+ override: override already rendered cluster manifests to run a local cluster-network-operator
+ attach:   attach to an existing cluster, stop cluster-network-operator, and restart with overrides
+ new:      install a new cluster using openshift-install and run the cluster-network-operator
+                  locally. This mode is assumed if the cluster-dir is not present or empty.
+
+The following option is required for all modes:
+
+ -c      the cluster installation temp directory; leave empty to create a cluster from scratch
+
+The following options are accepted for 'new' mode:
+
+ -f               the path to an openshift-install-created install-config.yaml file; if not given one will be created
+ -i               path to the openshift-install binary; if not given PATH will be searched
+ -m               custom network plugin container image name
+ -n [net plugin]  the name of the network plugin to deploy; one of [sdn|OpenShiftSDN|ovn|OVNKubernetes]
+
+The following environment variables are honored:
+ - CLUSTER_DIR: the cluster installation temp directory
```

Comment:
Above you say that `-c` is required... I don't think it's really that useful to allow both forms. (Also, the examples in HACKING.md are inconsistent.)

Reply:
@danwinship I think it's still useful, eg if you're going to use the same cluster temp dir a lot to re-test the CNO in a running cluster. But I'd probably just use -c for most stuff becuase the command-line looks better that way (to me) especially with CTRL+R. I updated the help text. PTAL.